e2e-tests/src/config.rs

```diff
@@ -87,13 +89,29 @@ impl Config {
             .unwrap()
     }
 
+    pub fn validator_names(&self) -> Vec<String> {
```

Comment:
Would parametrize this based on `validator_count`.

Reply:
done